recipes/setuptools-github/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "setuptools-github" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 51c0bd2fc6f4d703e87e0a78a97d6bc323a88d4559fc85598e44077e297368cb
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```

As a setuptools plugin, this needs `setuptools` installed to function properly, right? In that case you should have it here:
```suggestion
    - python >=3.6
    - setuptools
```